lib/launchers/process.js

```diff
@@ -64,11 +64,11 @@ var ProcessLauncher = function(spawn, tempDir, timer) {
 
     var errorOutput = '';
 
-    self._process.on('close', function(code) {
+    self._process.on('exit', function(code) {
       self._onProcessExit(code, errorOutput);
     });
 
-    self._process.on('error', function(err) {
+    self._process.on('karma_error', function(err) {
```

Comment:
You are totally right, thanks a lot fixed in https://github.com/karma-runner/karma/commit/45a69221703bbd043cd71b8b0934c4be5339e111